Import/HideTabsByIdentity/Import/Method/labs_HideTabsByIdentity.xml

```diff
@@ -7,127 +7,63 @@ var inn = document.thisItem.getInnovator();
-    }
-
+   
     //Check if relationship grid is loaded
     if (!isTabbarReady()) {
         setTimeout(hideTabs, 50);
         return;
     }
-
     tabbar = parent.relationships.relTabbar;
     desiredTabId = tabbar.GetTabId("Alternates");
     tabbar.selectTab(desiredTabId);
-
     //Check if desired tab is selected
     if (!isTabSelected(desiredTabId)) {
         setTimeout(hideTabs, 50);
         return;
     }
-
     var bomTab = tabbar.GetTabId("BOM");
     var mbomTab = tabbar.GetTabId("BOM Structure");
     tabbar.SetTabVisible(bomTab, false);
     tabbar.SetTabVisible(mbomTab, false);
-
     parent.relationshipsControl.onloaded();
 };
 
-
 // Function to check if relationship grid is loaded
 var isTabbarReady = function() {
     if (parent.relationships === undefined)
         return false;
-
     var isReady = Boolean(!parent.relationships ||
         !parent.relationships.relTabbarReady ||
         parent.relationships.relTabbar.GetTabOrder("|") !== "");
     return isReady;
 };
 
-
 // Function to check if desired tab is selected
 var isTabSelected = function(palceholderTabId) {
     return (tabbar.GetSelectedTab() === palceholderTabId);
 };
 
 // Function to check whether user belongs to an identity
-var memberCheck = function(identity, userName, levels) {
+var memberCheck = function(identity) {
     var isMember = 0;
-    var members = identity.getRelationships("Member");
-
-    if (!members.getItemCount() > 0)
-        return isMember;
-
-    for (var i = 0; i < members.getItemCount(); i++) {
-
-        if (isMember == 1)
-            return isMember;
-
-        if (isMember == -1)
-            return isMember;
-
-        var mem = members.getItemByIndex(i);
-        if (mem.getRelatedItem().getProperty("is_alias", "") == '0') {
-
-            var ident = inn.newItem("Identity", "get");
-            ident.setProperty("id", mem.getRelatedItem().getProperty("id", ""));
-            var memrel = ident.createRelationship("Member", "get");
-            ident = ident.apply();
+    var targetIdentity = inn.getItemByKeyedName("Identity", identity);
 
-            if (ident.isError())
-                return ident;
-
-            levels++;
-
-            if (levels > 9) //if membership > 10 levels possible circular reference breaks out of code
-                return -1;
-
-            isMember = memberCheck(ident, userName, levels);
-
-        } else {
-
-            idenName = mem.getRelatedItem().getProperty("keyed_name", "");
-            if (idenName == userName) {
-                isMember = 1;
-                break;
-            } else {
-                continue;
-            }
-        }
-    }
+    // getIdentitiyList returns the Identities to which the current users belongs (see programmers guide)
+    // -1 indicates, that the current user is not a member of the target identity 
+    isMember = aras.getIdentityList().indexOf(targetIdentity.getID());
```

Comment:
```suggestion
    isMember = aras.getIdentityList().includes(targetIdentity.getID());
```
Simplifying code is a great reason for a PR, and this is a nice way of showing off a useful API call in the process. But I think we can make this even more clear by using **array.includes** instead of **array.indexOf**. Calling the includes function will return a true or false value, which means at the top of the method you can simply check for **if(isMember)**.